modules/tinymce/src/core/main/ts/api/dom/Serializer.ts

```diff
@@ -105,7 +105,45 @@ const DomSerializer = (settings: DomSerializerSettings, editor?: Editor): DomSer
 
     getNodeFilters: domSerializer.getNodeFilters,
 
-    getAttributeFilters: domSerializer.getAttributeFilters
+    getAttributeFilters: domSerializer.getAttributeFilters,
+
+    /**
+     * Removes a node filter function or removes all filter functions from the parser used by the serializer for the node names provided.
+     *
+     * @method removeNodeFilter
+     * @param {String} name Comma separated list of node names to remove filters for.
+     * @param {Function} callback Optional callback function to only remove a specific callback.
+     * @example
+     * // Remove a single filter
+     * serializer.removeNodeFilter('p,h1', (nodes, name) => {
```

Comment:
This example makes little sense. Maybe just have it be `serializer.removeNodeFilter('p,h1', someFunction)` or maybe just remove the example here since this is not very commonly used. I tend to add examples to things that are used a lot.

Reply:
Yeah I did wonder that myself when creating these. I'll swap it to basically what you suggested, though I might use `someCallback` just to match the parameter name.